kin-devplatform-sdk/src/main/java/kin/devplatform/exception/MigrationNeededExceptions.java

```diff
@@ -0,0 +1,10 @@
+package kin.devplatform.exception;
+
+public class MigrationNeededExceptions extends BlockchainException{
+
+    public static final String EXCEPTION_MESSAGE = "sdk and server are not in the same blockchain version so migration is needed";
```

Comment:
I think it will be clearer to tell that the server is migrated to the new blockchain so the client must migrate as well

Reply:
Fix it, now it is better or to change it a bit?